src/resources/batches/BatchList.tsx

```diff
@@ -61,7 +61,7 @@ interface PlatformFilterType {
 }
 
 const PlatformFilter = (props: PlatformFilterType): React.ReactElement => {
-  const { data: batches } = useGetList('batches')
+  const { data: batches } = useGetList('batch')
```

Comment:
Hello @TahaKhanAbdalli - the cause of this issue is that we weren't referring to the `R_BATCHES` const.  Let's use it here, please.